tasks/install.yml

```diff
@@ -1,13 +1,34 @@
 ---
 
-- import_tasks: user.yml
+- name: Check presence of fastestmirror.conf

```

Comment:
I like your decision to speed up the deployment time, but first of all, Debian-like distros don't contain this file.
Second, such tasks should be used in any preliminary roles like `common`, `initial` and the like to prepare your system and do some basic initial steps.

Reply:
Added when: ansible_os_family == "RedHat"